src/Legacy/Drupal7.php

```diff
@@ -43,7 +43,14 @@
 class Drupal7 {
 
   public function __call($method, $args) {
-    return call_user_func_array($method, $args);
+    // Hack: See http://goo.gl/aM5fra
+    $Args = array();
+    foreach($args as $k => &$arg){
+      $Args[$k] = &$arg;
+    }
```

Comment:
I have used this hack myself before! :+1: I love it :-D.

Lets use $args_reference though instead of $Args and $key => &$arg.